presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -232,12 +264,10 @@ class EventViewModel @Inject constructor(
                             isGoing = _attendee.isGoing
                         )
 
-                        _eventScreenState.update { eventScreenState ->
-                            eventScreenState.copy(
-                                isEmailVerified = true,
-                                attendees = eventScreenState.attendees + attendee
-                            )
-                        }
+                        onEventScreenEvent(EventScreenEvent.OnAttendeeAdded(attendee))
+                        onEventScreenEvent(EventScreenEvent.OnVerifyingVisitorEmail(false))
+                        onEventScreenEvent(EventScreenEvent.OnShowVisitorDialog(false))
```

Comment:
If from the viewmodel you need to call an event, that´s not being called on the UI, then no need to create it as an event
It can just be a private fun